Update Composer

aliciakrebs opened the pull request:
<!-- Thanks for sending us a PR to improve this project! If you are adding a
feature or fixing a bug, and this needs more documentation, please add it to your PR.

**Thank you for contributing to the `sauce-docs` project!**
**A well described pull request helps maintainers quickly review and merge your change**

Before submitting your PR, please check our [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md)
guidelines, applied for this repository. Avoid large PRs, help reviewers by making them as simple
and short as possible. -->


<!--- Provide a general summary of your changes in the Title above -->

### Description
Update screenshots and content for new Composer.

### Motivation and Context

https://saucedev.atlassian.net/browse/TCD-2570


### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
<!--- Provide a general summary of your changes in the Title above -->

LoryApiFortress commented on docs/api-testing/composer.md:
```diff
@@ -579,59 +579,6 @@ To create an integration test to test the interaction between the endpoints:
 
     <img src={useBaseUrl('/img/api-testing/int-test-response-payload.png')} alt="Testing the response payload"/>
 
```
@aliciakrebs Replace the old version in xml with the new version in yaml
```yaml
- id: get
  children:
    - id: header
      name: key
      value: ABC123
  url: http://demoapi.apifortress.com/api/retail/product
  var: productsPayload
  mode: json
- id: if
  children:
    - id: comment
      text: endpoint is not working fine, test will be stopped
    - id: flow
      command: stop
  expression: productsPayload_response.statusCode!='200'
- id: assert-is
  expression: productsPayload
  comment: payload must be an array
  type: array
- id: each
  children:
    - id: comment
      text: "product id is: ${_1.id} and product name is: ${_1.name}"
    - id: assert-is
      expression: _1.id
      comment: id must be an integer value
      type: integer
    - id: set
      var: id
      mode: string
      value: ${_1.id}
    - id: assert-exists
      expression: _1.name
      comment: name must exists
    - id: assert-is
      expression: _1.price
      comment: price must be a float number
      type: float
    - id: assert-exists
      expression: _1.category
      comment: category must exists
    - id: assert-exists
      expression: _1.description
      comment: description must exists
    - id: assert-is
      expression: _1.quantity
      comment: quantity must be an integer value
      type: integer
    - id: assert-greater
      expression: _1.quantity
      comment: quantity must be greater than 0
      value: 0
    - id: assert-is
      expression: _1.imageURL
      comment: imageURL must be a valid url value
      type: url
    - id: assert-is
      expression: _1.color
      comment: color must be an array
      type: array
    - id: each
      children:
        - id: assert-exists
          expression: _2
          comment: color array should contain some values
        - id: assert-in
          expression: _2
          comment: colors must be the expected one
          value:
            - yellow
            - blue
            - red
            - green
            - brown
            - orange
            - gray
            - pink
            - black
            - white
      expression: _1.color
    - id: assert-exists
      expression: _1.createdAt
      comment: createdAt must exists
    - id: assert-exists
      expression: _1.updatedAt
      comment: updateAt must exists
    - id: comment
      text: get product details
    - id: get
      children:
        - id: header
          name: key
          value: ABC123
      url: http://demoapi.apifortress.com/api/retail/product/${id}
      var: productPayload
      mode: json
    - id: if
      children:
        - id: comment
          text: endpoint is not working fine, test will be stopped
        - id: flow
          command: stop
      expression: productPayload_response.statusCode!='200'
    - id: assert-exists
      expression: productPayload
      comment: payload must exist, if not, test does not need to be executed
    - id: comment
      text: "product id is: ${productPayload.id} and product name is:
        ${productPayload.name}"
    - id: assert-is
      expression: productPayload.id
      comment: id must be an integer value
      type: integer
    - id: assert-exists
      expression: productPayload.name
      comment: name must exists
    - id: assert-is
      expression: productPayload.price
      comment: price must be a float number
      type: float
    - id: assert-exists
      expression: productPayload.category
      comment: category must exists
    - id: assert-exists
      expression: productPayload.description
      comment: description must exists
    - id: assert-is
      expression: productPayload.quantity
      comment: quantity must be an integer value
      type: integer
    - id: assert-greater
      expression: productPayload.quantity
      comment: quantity must be greater than 0
      value: 0
    - id: assert-is
      expression: productPayload.imageURL
      comment: imageURL must be a valid url value
      type: url
    - id: assert-is
      expression: productPayload.color
      comment: color must be an array
      type: array
    - id: each
      children:
        - id: assert-exists
          expression: _2
          comment: color array should contain some values
        - id: assert-in
          expression: _2
          comment: colors must be the expected one
          value:
            - yellow
            - blue
            - red
            - green
            - brown
            - orange
            - gray
            - pink
            - black
            - white
      expression: productPayload.color
    - id: assert-exists
      expression: productPayload.createdAt
      comment: createdAt must exists
    - id: assert-exists
      expression: productPayload.updatedAt
      comment: updateAt must exists
  expression: productsPayload.pick(5)
```

LoryApiFortress commented on docs/api-testing/composer.md:
````diff
@@ -662,12 +609,16 @@ You can create specific assertions to verify performance metrics.
-  <assert-less expression="payload_response.metrics.fetch" value="350" type="integer"/>
-
-  <assert-less expression="payload_response.metrics.overall" value="550" type="integer"/>
+  ```yaml
````
@aliciakrebs please pay attention to how you write the code because in yaml spaces and indentation are important

```yaml
- id: assert-less
   expression: payload_response.metrics.latency
   value: 350
- id: assert-less
   expression: payload_response.metrics.fetch
   value: 350
- id: assert-less
   expression: payload_response.metrics.overall
   value: 450
```

LoryApiFortress reviewed (commented):
@aliciakrebs it's not a big issue but there are some images that are not completely correct:
- [int-getting-token.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-03dd7e5833673948d7836e5edde4ef15219f300bdd1a0def88da6f9c154c3f2c)
- [int-assign-token.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-e3474909df5a6d65a3e78d2f3ba8a60609be55f52e11bac84842aa1cc740e841)
- [int-reuse-tokens.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-9de0d1cdc49622de336df58e238b9c2c5159c295a1d71ccc0a1c0fe40dc32c17)
- [int-set-token-header.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-a6e6a3eeef89751d96576f3569e3175b628df528e4aff8d8188c2e52e4e7f062)
In the new images, the body contains "access_token" but since the request should be to "request the token" it does not make sense to add in the body something you want to retrieve making the request.

aliciakrebs commented:
> @aliciakrebs it's not a big issue but there are some images that are not completely correct:
> 
> * [int-getting-token.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-03dd7e5833673948d7836e5edde4ef15219f300bdd1a0def88da6f9c154c3f2c)
> * [int-assign-token.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-e3474909df5a6d65a3e78d2f3ba8a60609be55f52e11bac84842aa1cc740e841)
> * [int-reuse-tokens.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-9de0d1cdc49622de336df58e238b9c2c5159c295a1d71ccc0a1c0fe40dc32c17)
> * [int-set-token-header.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-a6e6a3eeef89751d96576f3569e3175b628df528e4aff8d8188c2e52e4e7f062)
>   In the new images, the body contains "access_token" but since the request should be to "request the token" it does not make sense to add in the body something you want to retrieve making the request.



> @aliciakrebs it's not a big issue but there are some images that are not completely correct:
> 
> * [int-getting-token.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-03dd7e5833673948d7836e5edde4ef15219f300bdd1a0def88da6f9c154c3f2c)
> * [int-assign-token.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-e3474909df5a6d65a3e78d2f3ba8a60609be55f52e11bac84842aa1cc740e841)
> * [int-reuse-tokens.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-9de0d1cdc49622de336df58e238b9c2c5159c295a1d71ccc0a1c0fe40dc32c17)
> * [int-set-token-header.png](https://github.com/saucelabs/sauce-docs/pull/1532/files#diff-a6e6a3eeef89751d96576f3569e3175b628df528e4aff8d8188c2e52e4e7f062)
>   In the new images, the body contains "access_token" but since the request should be to "request the token" it does not make sense to add in the body something you want to retrieve making the request.

@LoryApiFortress Thank you for letting me know. I have fixed this, and addressed the other comments as well.
